### PR TITLE
How to update the tWAS on Azure VMs solution for next tWAS fixpack

### LINKED ISSUE #30: Create document(wiki page) to update the solution for next WAS fixpack
The document should contain following information:
- Updating the image
    - Which file to update for WAS version?
    - How to run CI/CD?
    - How to test the image, what testcases to run?
    - How to publish the image in marketplace and who can do it?
    - Do we need to update the solution every time we do the image update?
- Updating and publishing the solution code
    - This section should also include steps to make any bug fixes (without any changes to the image) and publishing it.
    - How to test the solution, what testcases to run?
    - How to publish the solution in marketplace and who can do it?
- What needs to be cleaned up from test env and how to clean them up?
- Do we delete/archive previous version of the solution?
- Create a release and a branch with the GA code (for image and cluster repo)

### PR DESCRIPTION
**Pull request by majguo**
The PR is to fix #30.

Signed-off-by: Jianguo Ma <jiangma@microsoft.com>

### CODE REVIEW

File: docs/howto-update-for-was-fixpack.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+1. Which file to update for WAS version?
+   * For `twas-nd` image, update the following properties in file [`virtualimage.properties`](https://github.com/WASdev/azure.websphere-traditional.image/blob/master/twas-nd/src/main/scripts/virtualimage.properties#L14-L15), e.g.:
+
+     ```bash
+     WAS_ND_TRADITIONAL=com.ibm.websphere.ND.v90
+     IBM_JAVA_SDK=com.ibm.java.jdk.v8
+     ```
+
+     Note: only the major version should be specified, the minor version should not be hard-coded as the Installation Manager will intelligently install the latest available minior version.
```
Comment (git4rk) [inline docs/howto-update-for-was-fixpack.md:15]: minior -> minor

File: docs/howto-update-for-was-fixpack.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+   * For `ihs` image, update the following properties in file [`virtualimage.properties`](https://github.com/WASdev/azure.websphere-traditional.image/blob/master/ihs/src/main/scripts/virtualimage.properties#L22-L25), e.g.:
+
+     ```bash
+     IBM_HTTP_SERVER=com.ibm.websphere.IHS.v90
+     WEBSPHERE_PLUGIN=com.ibm.websphere.PLG.v90
+     WEBSPHERE_WCT=com.ibm.websphere.WCT.v90
+     IBM_JAVA_SDK=com.ibm.java.jdk.v8
+     ```
+
+     Note: only the major version should be specified, the minor version should not be hard-coded as the Installation Manager will intelligently install the latest available minior version.
```
Comment (git4rk) [inline docs/howto-update-for-was-fixpack.md:26]: minior -> minor

File: docs/howto-update-for-was-fixpack.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+   1. For `ihs` image: Wait until the CI/CD workflow for `ihs CICD` successfully completes > Click to open details of the workflow run > Scroll to the bottom of the page > Click `sasurl` to download the zip file `sasurl.zip` > Unzip and open file `sas-url.txt` > Find values for `osDiskSasUrl` and `dataDiskSasUrl`;
+   1. Sign into [Microsoft Partner Center](https://partner.microsoft.com/dashboard/commercial-marketplace/overview) > Click to open the offer for `twas-nd` base image > Clic `Plan overview` > Click to open the plan > Clic `Technical configuration` > Click `+ Add VM image` > Specify a new value for `Disk version` and write it down > Select `SAS URI` > Copy and paste value of `osDiskSasUrl` for `twas-nd` to textbox `SAS URI` > Click `+ Add data disk (max 16)` > Select `Data disk 0` > Copy and paste value of `dataDiskSasUrl` for `twas-nd` to textbox `Data disk VHD link` > Scroll to the bottom of the page > Click `Save draft` > Click `Review and publish` > Click `Publish`;
```
Comment (git4rk) [inline docs/howto-update-for-was-fixpack.md:38]: Clic -> Click

File: docs/howto-update-for-was-fixpack.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+   1. Sign into [Microsoft Partner Center](https://partner.microsoft.com/dashboard/commercial-marketplace/overview) > Click to open the offer for `ihs` base image > Clic `Plan overview` > Click to open the plan > Clic `Technical configuration` > Click `+ Add VM image` > Specify a new value for `Disk version` and write it down > Select `SAS URI` > Copy and paste value of `osDiskSasUrl` for `ihs` to textbox `SAS URI` > Click `+ Add data disk (max 16)` > Select `Data disk 0` > Copy and paste value of `dataDiskSasUrl` for `ihs` to textbox `Data disk VHD link` > Scroll to the bottom of the page > Click `Save draft` > Click `Review and publish` > Click `Publish`;
```
Comment (git4rk) [inline docs/howto-update-for-was-fixpack.md:39]: Clic -> Click

File: docs/howto-update-for-was-fixpack.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+   Note: Currently Graham Charters has privilidge to update the image in marketplace, contact him for more information.
```
Comment (git4rk) [inline docs/howto-update-for-was-fixpack.md:41]: privilidge -> privilege

File: docs/howto-update-for-was-fixpack.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+1. How to publish the solution in marketplace and who can do it?
+   1. Wait until the CI/CD workflow for `Package ARM` successfully completes > Click to open details of the workflow run > Scroll to the bottom of the page > Click `azure.websphere-traditional.cluster-<version>-arm-assembly` to download the zip file `azure.websphere-traditional.cluster-<version>-arm-assembly.zip`;
+   1. Sign into [Microsoft Partner Center](https://partner.microsoft.com/dashboard/commercial-marketplace/overview) > Click to open the offer for the solution > Clic `Plan overview` > Click to open the plan > Clic `Technical configuration` > Specify the increased version number for `Version` > Click `Remove` to remove the previous package file > Click `browse your file(s)` to upload the downloaded zip package generated by the CI/CD pipeline before > Scroll to the bottom of the page > Click `Save draft` > Click `Review and publish` > Click `Publish`;
+
+   Note: Currently Graham Charters has privilidge to update the image in marketplace, contact him for more information.
```
Comment (git4rk) [inline docs/howto-update-for-was-fixpack.md:60]: privilidge -> privilege

File: docs/howto-update-for-was-fixpack.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+1. How to test the solution, what testcases to run?
+   1. Wait until the soluton offer is in `Publisher signoff` (aka "preview") stage;
+   1. Run test cases defined in [twas-solutoin-test-cases.pdf](twas-solutoin-test-cases.pdf). Note: use "preview link" for each test case.
+
+## What needs to be cleaned up from test env and how to clean them up?
+
+Azure marketplance is responsible for managing different stages during the offer publishing, just follow its process to make it Go-Live and no additional clean-ups are neeed.
```
Comment (git4rk) [inline docs/howto-update-for-was-fixpack.md:68]: marketplance -> marketplace

File: docs/howto-update-for-was-fixpack.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+   1. Sign into [Microsoft Partner Center](https://partner.microsoft.com/dashboard/commercial-marketplace/overview) > Click to open the offer for the solution > Clic `Plan overview` > Click to open the plan > Clic `Technical configuration` > Specify the increased version number for `Version` > Click `Remove` to remove the previous package file > Click `browse your file(s)` to upload the downloaded zip package generated by the CI/CD pipeline before > Scroll to the bottom of the page > Click `Save draft` > Click `Review and publish` > Click `Publish`;
+
+   Note: Currently Graham Charters has privilidge to update the image in marketplace, contact him for more information.
+
+1. How to test the solution, what testcases to run?
+   1. Wait until the soluton offer is in `Publisher signoff` (aka "preview") stage;
+   1. Run test cases defined in [twas-solutoin-test-cases.pdf](twas-solutoin-test-cases.pdf). Note: use "preview link" for each test case.
```
Comment (git4rk) [inline docs/howto-update-for-was-fixpack.md:64]: twas-solutoin-test-cases.pdf -> twas-solution-test-cases.pdf
Fix the file name of the pdf file also.

File: docs/howto-update-for-was-fixpack.md (hunk 1 above)
Comment (majguo) [reply docs/howto-update-for-was-fixpack.md]: Fixed in 7c7bef3.

File: docs/howto-update-for-was-fixpack.md (hunk 2 above)
Comment (majguo) [reply docs/howto-update-for-was-fixpack.md]: Fixed in 7c7bef3.

File: docs/howto-update-for-was-fixpack.md (hunk 3 above)
Comment (majguo) [reply docs/howto-update-for-was-fixpack.md]: Fixed in 7c7bef3.

File: docs/howto-update-for-was-fixpack.md (hunk 4 above)
Comment (majguo) [reply docs/howto-update-for-was-fixpack.md]: Fixed in 7c7bef3.

File: docs/howto-update-for-was-fixpack.md (hunk 5 above)
Comment (majguo) [reply docs/howto-update-for-was-fixpack.md]: Fixed in 7c7bef3.

File: docs/howto-update-for-was-fixpack.md (hunk 6 above)
Comment (majguo) [reply docs/howto-update-for-was-fixpack.md]: Fixed in 7c7bef3.

File: docs/howto-update-for-was-fixpack.md (hunk 7 above)
Comment (majguo) [reply docs/howto-update-for-was-fixpack.md]: Fixed in 7c7bef3.

File: docs/howto-update-for-was-fixpack.md (hunk 8 above)
Comment (majguo) [reply docs/howto-update-for-was-fixpack.md]: Fixed in 7c7bef3.

**Review by git4rk — APPROVED** [review]

**Pull request merged**